### ISSUE TITLE
dumb terminal

### ISSUE THREAD

**Issue by fommil** [drone/drone#1048]
As discussed on gitter and http://stackoverflow.com/questions/30674922/ my builds need access to a tty (even though it is not used). It appears that passing `-t` to docker should make this work.

I'd be happy to test a self-built cut of drone with this change if that helps. Just let me know the commit to cherry pick on top of the last stable (or give me a branch that I can build... assuming you don't have a CI job for `.deb` snapshot files, which would be awesome btw!)

**Comment by bradrydzewski** [comment]
@fommil this is the flag that needs to be enabled:
https://github.com/drone/drone/blob/master/shared/build/docker/structs.go#L128

which could be set here:
https://github.com/drone/drone/blob/master/shared/build/docker/structs.go#L128

we already enable setting certain Docker parameters in the yaml, such as hostname and a few others. We could do this with tty. It would be false by default, but could be enabled:

```yaml
docker:
  tty: true
```

we do auto-generate `.deb` files as part of our build and push to S3 once the change is merged into master.

note that this change would need to get ported to 0.4 still, but perhaps the above instructions would let you proceed with your project. cheers!

**Comment by fommil** [comment]
Awesome thanks, I'm at a conference this week but will try this out on a local build.

**Comment by fommil** [comment]
@bradrydzewski was this ever ported into the current "stable" deb release?
